tests/e2e/csi_static_provisioning_basic.go

```diff
@@ -1451,22 +1450,21 @@ var _ = ginkgo.Describe("Basic Static Provisioning", func() {
 		time.Sleep(time.Duration(pandoraSyncWaitTime) * time.Second)
 
 		ginkgo.By("Create CNS register volume with above created FCD")
-		log.Infof("Auto created namespace :%s", autocreatednamespace)
+		framework.Logf("Auto created namespace :%s", autocreatednamespace)
 		cnsRegisterVolume := getCNSRegisterVolumeSpec(ctx, autocreatednamespace, fcdID, "", pvcName, v1.ReadWriteOnce)
 		err = createCNSRegisterVolume(ctx, restConfig, cnsRegisterVolume)
 		gomega.Expect(err).NotTo(gomega.HaveOccurred())
-		time.Sleep(time.Duration(40) * time.Second)
+		time.Sleep(time.Duration(pollTimeoutShort))
```

Comment:
should we add polling instead of static sleep?

Reply:
Added polling

Reply:
Done